README.md

```diff
@@ -83,16 +85,20 @@ An example run of this action might look like this:
       controller=lxd
 
 Running this action will result in a new job in Jenkins called
-`charm-<charmname>` that will poll the repository once every 5 minutes. In case
-new commits are found, bundletester will test your charm and the result will be
-pushed to the store (again, given you have opted in).
+`charm-<charmname>`. This job is triggered by a webhook that is available in the
+output of the action. You will need to add this webhook under Settings->Webhooks
+of your github repository.
```

Comment:
We probably need to add a more thorough description of how webooks work to the README, or in this help text.

It's not clear to me what url I need to enter in github for my webhook (presumably something like http://<this charm's ip>/some/jenkins/url).

This also adds some complications when it comes to backup/restore, correct?